app/controllers/people/people_skills_controller.rb

```diff
@@ -32,8 +32,21 @@ def new
 
   def update
     @people_skills = filtered_people_skills
+    if params['people_skill'][:level] == '0'
+      params['people_skill'][:unrated] = true
```

Comment:
I don't understand why you set a parameter based on another parameter. And what is it used for?

Reply:
Okay sure but in the rest of the application you can't set it back to `0` after having rated it.
E.g. when you list rated skills you can't set them to 0...
Therefore it probably makes more sense to remove the option to set it to `0` after having rated it.